demos/python_demos/common/images_capture.py

```diff
@@ -0,0 +1,114 @@
+import os
+import copy
+
+import cv2
+
+
+class InvalidInput(Exception):
+    pass
+
+
+class ImagesCapture():
+
+    def __init__(self, loop):
+        self.loop = loop
+        self.isVideo = False
```

Comment:
Demos could test if `openImagesCapture()` returned an instance of `VideoReader` instead.

Reply:
Done